### DISCUSSION TITLE
Is it possible to upgrade to the new 24.04 LTS from the previous 22.04?

### DISCUSSION THREAD

**Discussion by JanV123** [Joshua-Riek/ubuntu-rockchip#764] [Q&A] [answered]
Hi. I apologize in advance if this is a dumb question.
I'm currently running my OP5 as a Pi-Hole and TailScale site-to-site server. I'm running the 22.04 LTS on it and I was wondering if it's possible to upgrade to the new 24.04 LTS without having to wipe everything? I tried to do a `sudo do-release-upgrade -d` which is what I normally do for standard x86 Ubuntu machines but it doesn't work.

**Comment by Joshua-Riek** [comment, marked as answer]
Sadly, the short answer is no. 

There are many system level changes that would make this very challenging to provide a smooth upgrade process. Additionally, since this is a hobby project and I'm not paid by any corporate entities I can't invest the time required to properly provide users with an upgrade path from Ubuntu 22.04 to Ubuntu 24.04.

**Reply by JanV123** [reply]
Thanks for the quick reply! It's completely understandable. It was worth asking. Once upgraded to 24.04 LTS, is it safe to assume that succeeding 24.04 LTS releases (ie. v2.1.0) you make will be available through the standard `sudo apt dist-upgrade` method or is it required to flash the nvme drive each time?

**Reply by Joshua-Riek** [reply]
All updates and changes will be available through an apt upgrade. I added some additional packages to assist with this process in Ubuntu 24.04.

**Reply by JanV123** [reply]
Thanks again for the replies and your hard work.

**Reply by ljuEUCL** [reply]
> All updates and changes will be available through an apt upgrade. I added some additional packages to assist with this process in Ubuntu 24.04.

Does this mean that it is posible to update to 24.04 (from 22.04) by replacing the apt sources and then apt upgrade the manual way?
